### ISSUE TITLE
After updating to Flutter 3.10 alert shows at top of screen

### ISSUE THREAD

**Issue by bermanapps** [RatelHub/rflutter_alert#139]
Hi
After updating to Flutter 3.10, the alert now shows up in the top of screen, not in the center. 
Why would that be, and how can I solve that?
Thanks!

**Comment by luis-rodas** [comment]
Yes, agree with @bermanapps I did just update to lasted release now show in top screen

In your sample:

![Screenshot 2023-05-18 at 10 45 39](https://github.com/RatelHub/rflutter_alert/assets/90215451/bf26fec2-e12a-4727-9ead-296bcfb47bb1)

**Comment by luis-rodas** [comment]
The problem comes from this commit:
https://github.com/RatelHub/rflutter_alert/commit/ff742e772194c3d0abdaaee431dbb879f19d25c8


If it reverted it's works fine

![Screenshot 2023-05-18 at 11 16 35](https://github.com/RatelHub/rflutter_alert/assets/90215451/fd62a5eb-97dd-4dae-9a0e-32dd314cf193)

**Comment by luis-rodas** [comment]
I think I fixed, now you need to pass the box constrain in style property:
Like that:

```
var alertStyle = AlertStyle(
    constraints: BoxConstraints(minHeight: 200),
  );

// The easiest way for creating RFlutter Alert
  _onBasicAlertPressed(context) {
    Alert(
      style: alertStyle,
      context: context,
      title: "RFLUTTER ALERT",
      desc: "Flutter is more awesome with RFlutter Alert.",
    ).show();
  }
```

<img width="795" alt="Screenshot 2023-05-18 at 12 02 28" src="https://github.com/RatelHub/rflutter_alert/assets/90215451/a5a74794-36ed-47e2-84fc-b1375cd6ca99">

**Comment by iulian0512** [comment]
i confirm this issue

**Comment by ibrahimdevs** [comment]
Everyone dealing with this issue, it was about to default constraint and fixed in v2.0.7
